src/models/generated/DER_A.jl

```diff
@@ -0,0 +1,160 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct DER_A <: InnerControl
```

Comment:
The names of struct can't have `_`. We need to pick something more descriptive like `DistributedResourceA` or `ActiveDistributionLoadA`

Reply:
My vote goes to ActiveDistributionLoadA or ActiveDistributionNetworkA. I prefer LoadA.

Reply:
This is a `<: DynamicInjection`